From 9338bcf0a033a4d5055d663885c086feaa078337 Mon Sep 17 00:00:00 2001
From: Ben Kircher <bkircher@0xadd.de>
Date: Mon, 21 Feb 2022 16:24:02 +0100
Subject: [PATCH] Backport Elasticsearch test fixes from master

Fix out_elasticsearch tests by backporting relevant changes from master
branch.

diff --git a/tests/runtime/out_elasticsearch.c b/tests/runtime/out_elasticsearch.c
index d420e11dd..886f8ed05 100644
--- a/tests/runtime/out_elasticsearch.c
+++ b/tests/runtime/out_elasticsearch.c
@@ -13,7 +13,7 @@ static void cb_check_index_type(void *ctx, int ffd,
 {
     char *p;
     char *out_js = res_data;
-    char *index_line = "{\"index\":{\"_index\":\"index_test\",\"_type\":\"type_test\"}";
+    char *index_line = "{\"create\":{\"_index\":\"index_test\",\"_type\":\"type_test\"}";
 
     p = strstr(out_js, index_line);
     TEST_CHECK(p != NULL);
@@ -27,7 +27,7 @@ static void cb_check_logstash_format(void *ctx, int ffd,
 {
     char *p;
     char *out_js = res_data;
-    char *index_line = "{\"index\":{\"_index\":\"prefix-2015-11-24\",\"_type\":\"_doc\"}";
+    char *index_line = "{\"create\":{\"_index\":\"prefix-2015-11-24\",\"_type\":\"_doc\"}";
 
     p = strstr(out_js, index_line);
     TEST_CHECK(p != NULL);
-- 
2.35.1

